src/DwitTech.AccountService.Data/Entities/ValidationCode.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.ComponentModel.DataAnnotations;
+using System.Linq;
+using System.Text;
+using System.Threading.Channels;
+using System.Threading.Tasks;
+using DwitTech.AccountService.Core;
+using DwitTech.AccountService.Core.Enums;
+using Microsoft.EntityFrameworkCore;
+
+namespace DwitTech.AccountService.Data.Entities
+{
+    public class ValidationCode : BaseEntity
```

Comment:
Since this entity inherits from base entity, please change "{set;}" to "{set; private get;}" in the CreatedOnUtc property of BaseEntity before running your migration, otherwise, the CreatedOnUtc value likely won't show up as a ValidationCode column on the migration sheet, and it will affect the table too.

Reply:
@giftedGeorge doesn't PR  #11 solve this? If that's the case, no need duplicating effort.

Reply:
Yes boss, but if she's locally generating migrations for an entity that inherits from base entity, without the solution, isn't she going to encounter the same issue I did without knowing there's a problem with her migration?